tests/sequential/opentelemetry_metrics/opentelemetry_metrics_test.go

```diff
@@ -38,6 +38,7 @@ var (
 	wrongScalerName                          = fmt.Sprintf("%s-wrong-scaler", testName)
 	cronScaledJobName                        = fmt.Sprintf("%s-cron-sj", testName)
 	clientName                               = fmt.Sprintf("%s-client", testName)
+	kedaOperatorPrometheusURL                = "http://keda-operator.keda.svc.cluster.local:8080/metrics"
```

Comment:
You have to get this metric from current collector url, if you check the KEDA's endpoint, it doesn't check if the metric is sent via otel

Reply:
How can I get the current collector url

Reply:
I have updated the function to use this collector url 
`kedaOperatorCollectorPrometheusExportURL`